### PR TITLE
Sadtalker triton bgremoval

### PR DESCRIPTION
**Pull request by pritishaim**
Please check

### CODE REVIEW

File: src/bgremoval_package/demo/run.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+torch_transforms = transforms.Compose(
+    [
+        transforms.ToTensor(),
+        transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5)),
+    ]
+)
+def load_model():
+    filepath = os.path.dirname(os.path.abspath(__file__))
+    ckpt_folder = Path(filepath).parent.parent.parent.parent
+    pretrained_ckpt = os.path.join(ckpt_folder,"checkpoints/bgremoval/modnet_webcam_portrait_matting.ckpt")
```
Comment (littleGiant-28) [inline src/bgremoval_package/demo/run.py]: Keep this string as global variable

File: src/bgremoval_package/demo/run.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    modnet = MODNet(backbone_pretrained=False)
+    modnet = nn.DataParallel(modnet)
+
+    GPU = True if torch.cuda.device_count() > 0 else False
```
Comment (littleGiant-28) [inline src/bgremoval_package/demo/run.py]: Also keep this as global variable

File: src/facerender/animate.py [hunk 3]
```diff
@@ -245,16 +246,20 @@ def generate(self, x, video_save_dir, pic_path, crop_info, enhancer=None, backgr
             av_path_enhancer = os.path.join(video_save_dir, video_name_enhancer) 
             return_path = av_path_enhancer
 
-            try:
-                enhanced_images_gen_with_len = enhancer_generator_with_len(full_video_path, method=enhancer, bg_upsampler=background_enhancer)
-                imageio.mimsave(enhanced_path, enhanced_images_gen_with_len, fps=float(25))
-            except:
-                enhanced_images_gen_with_len = enhancer_list(full_video_path, method=enhancer, bg_upsampler=background_enhancer)
+            # try:
```
Comment (littleGiant-28) [inline src/facerender/animate.py]: Is this try except logic surely not needed?

Comment (pritishaim) [reply src/facerender/animate.py]: Yes Initially the logic was to first return generator class output for video and if that fails then generate list of frames. , however we need the output as list of frames now which was initially in the except part hence i changed to always run the except part of the code.

File: src/bgremoval_package/demo/run.py (hunk 1 above)
Comment (pritishaim) [reply src/bgremoval_package/demo/run.py]: Ok

File: src/bgremoval_package/demo/run.py (hunk 2 above)
Comment (pritishaim) [reply src/bgremoval_package/demo/run.py]: Ok

File: src/facerender/animate.py (hunk 3 above)
Comment (littleGiant-28) [reply src/facerender/animate.py]: Ok

**Review by littleGiant-28 — APPROVED** [review]

**Pull request merged**